custom_components/tuya_local/devices/loratap_wifi_curtain_switch_double.yaml

```diff
@@ -0,0 +1,175 @@
+name: Loratap Wifi Double Curtain Switch
+products:
+  - id: zqypumy8ot4clzyi
+primary_entity:
+  entity: cover
+  name: Curtain
+  class: blind
+  dps:
+    - id: 1
+      name: control
+      type: string
+      mapping:
+        - dps_val: open
+          value: open
+        - dps_val: close
+          value: close
+        - dps_val: stop
+          value: stop
+    - id: 2
+      name: position
+      type: integer
+      unit: "%"
+      range:
+        max: 100
+        min: 0
+      mapping:
+        - step: 1
+secondary_entities:
+  - entity: cover
+    name: Curtain 2
+    dps:
+      - id: 4
+        name: control
+        type: string
+        mapping:
+          - dps_val: open
+            value: open
+          - dps_val: close
+            value: close
+          - dps_val: stop
+            value: stop
+      - id: 5
+        name: position
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - step: 1
+  - entity: number
+    name: Position 1
+    category: config
+    icon: "mdi:blinds-horizontal-closed"
+    dps:
+      - id: 2
+        name: value
+        type: integer
+        unit: "%"
+        range:
+          max: 100
+          min: 0
+        mapping:
+          - scale: 0
```

Comment:
This mapping is incorrect.  It contains two separate entries without a dps_val, but there can only be one default, and a scale of 0 will result in a division by zero error (tuya-local does not define scale in the same way as iot.tuya.com).   But since the whole entity is not needed, it can all be removed.